### PR TITLE
Update to latest rollup config changes.

### PR DESCRIPTION
**Pull request by jbucaran**
See https://gist.github.com/Rich-Harris/d472c50732dab03efeb37472b08a3f32

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ export default function(options = {}) {
                 code: `${codeExportSparse}export default ${injectFnName}(${JSON.stringify(
                   result.css
                 )},${JSON.stringify(codeExportDefault)});`,
-                map: options.sourceMap && result.map
+                map: options.sourcemap && result.map
```
Comment (egoist) [inline src/index.js]: maybe `options.sourcemap || options.sourceMap` to not break backward compatibility?

**Comment by egoist** [comment]
can you update rollup as well? seems current tests are failing because of that

**Comment by lmihaidaniel** [comment]
Also the [tests](https://github.com/egoist/rollup-plugin-postcss/blob/master/test/build.js) should be updated too.

**Comment by jbucaran** [comment]
@egoist and @lmihaidaniel, 

I tried updating the tests, but it seems rollup changes are more involved than I first thought. 

I got really close to a wrap, but the changes in how `.generate()` work set me back again so I'm going to call it a day for me and leave it up to you guys. 👋😄    

In the meanwhile, I'll keep using my patched version of plugin-postcss locally.

**Comment by lmihaidaniel** [comment]
here #44

**Pull request closed without merging**